src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -38,10 +39,16 @@ def environment_variables_format(value):
 register_cli_argument('container create', 'memory', type=float, help='The required memory of the containers in GB')
 register_cli_argument('container create', 'os_type', help='The OS type of the containers', **enum_choice_list(OS_TYPES))
 register_cli_argument('container create', 'ip_address', help='The IP address type of the container group', **enum_choice_list(IP_ADDRESS_TYPES))
+register_cli_argument('container create', 'ports', type=int, nargs='+', help='The ports to open')
```

Comment:
With this, I wouldn't be able to specify a port range.
Can you support that as part of this PR potentially?

Reply:
We only support max 5 ports. Opening port range probably doesn't make much sense at the moment?

Reply:
Then port range doesn't make sense.
The 5 port max is good to know and should be documented then in `help=`.
How come it isn't documented in swagger either?
This would mean that SDK users would not know this fact either.
https://github.com/Azure/azure-rest-api-specs/blob/current/specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-10-01-preview/containerInstance.json#L335

Reply:
Yea, I agree we need to document well such limits. Will loop in our PM on this.

Reply:
You can add a validator to this parameter that checks len(arg)<=5.